.github/workflows/nodejs-build.yaml

```diff
@@ -23,15 +23,23 @@ on:
         required: false
         type: string
         default: 'package'
-
+      node_version:
+        required: false
+        type: string
+      npm_version:
+        required: false
+        type: string
 jobs:
   build:
     runs-on: ubuntu-latest
     timeout-minutes: 10
     steps:
       - uses: actions/checkout@v3
       - name: Install dependencies
-        uses: dvsa/.github/.github/actions/install-deps@main
+        uses: dvsa/.github/.github/actions/install-deps@feature/VOL-3681-node-version-definable
```

Comment:
does this not just become a new version of install deps at this point .. I know a bit chicken and egg

Reply:
Pointing at branch during development for testing 😄